packages/xrpl/src/models/transactions/MPTokenIssuanceCreate.ts

```diff
@@ -150,12 +152,26 @@ export function validateMPTokenIssuanceCreate(
     }
   }
 
+  // eslint-disable-next-line @typescript-eslint/consistent-type-assertions -- Not necessary
+  const flags = tx.Flags as number | MPTokenIssuanceCreateFlagsInterface
+  const isTfMPTCanTransfer =
+    typeof flags === 'number'
+      ? isFlagEnabled(flags, MPTokenIssuanceCreateFlags.tfMPTCanTransfer)
+      : flags.tfMPTCanTransfer ?? false
+
   const MAX_TRANSFER_FEE = 50000
   if (typeof tx.TransferFee === 'number') {
```

Comment:
Should add a ValidationError when `TransferFee` isn't a `number`.

Reply:
don't think that's necessary since `TransferFee` only supports `number` type

Reply:
updated